package-lock.json

```diff
@@ -0,0 +1,9473 @@
+{
```

Comment:
Can this be removed? We're using yarn for dependency locking.

Reply:
I deleted the file `package-lock.json`

I would appreciate it if you could confirm this change.